packages/components/src/PieChartButton/PieChartButton.test.js

```diff
@@ -13,11 +13,90 @@ import {
 	wrapMouseEvent,
 } from './PieChartButton.component';
 
-initStoryshots({
-	configPath: 'src/PieChartButton/__snapshots__',
-});
-
 describe('PieChartButton', () => {
+	describe('snapshots render', () => {
+		const pieChartData = [
+			{
+				color: 'rio-grande',
+				percentage: 50,
+			},
+			{
+				color: 'chestnut-rose',
+				percentage: 12,
+			},
+			{
+				color: 'lightning-yellow',
+				percentage: 1,
+			},
+			{
+				color: 'slate-gray',
+				percentage: 4,
+			},
+			{
+				color: 'silver-chalice',
+				percentage: 3,
+			},
+		];
+
+		it('shoud render a PieChartButton', () => {
+			const onClick = jest.fn();
+			const event = {};
+			const wrapper = shallow(
+				<PieChartButton label="my label" display="small" model={pieChartData} onClick={onClick} />,
+			);
+
+			wrapper
+				.find('Button')
+				.at(0)
+				.simulate('click', event);
+
+			expect(onClick).toHaveBeenCalledWith(event, {
+				action: {
+					label: 'my label',
+				},
+				model: pieChartData,
+			});
+
+			expect(wrapper.getElement()).toMatchSnapshot();
+		});
+
+		it('shoud render a PieChartButton with an overlay', () => {
+			const onClick = jest.fn();
+			const event = {};
+			const overlayComponent = <div>I am an overlay</div>;
+			const wrapper = shallow(
+				<PieChartButton
+					display="medium"
+					labelIndex={2}
+					model={pieChartData}
+					overlayComponent={overlayComponent}
+					overlayId="id-popover"
+					onClick={onClick}
+				/>,
+			);
+
+			wrapper
+				.find('Button')
+				.at(0)
+				.simulate('click', event);
+
+			expect(onClick).not.toHaveBeenCalledWith(event, {
```

Comment:
Same here,
* title is `should render a PieChartButton with an overlay`
* you test a click, that should not trigger onClick
Not really clear in my head